theme/home.less

```diff
@@ -173,6 +202,12 @@
     .import-dialog-btn {
         position: relative;
         z-index: 1; /* Move up so it's above the carousel container that has an offset margin */
+        &:focus-visible {
+            outline-width: @homeCardBorderSize !important;
+            outline-style: solid;
+            outline-color: var(--pxt-primary-background);
```

Comment:
For this one, I'm wondering if var(--pxt-focus-border) would be a more appropriate color. Would you be open to trying that and seeing if it still aligns to standards?

If we add more themes in the future, the primary color may vary, but the "focus border" should remain a fairly blue across all themes.

It makes sense to deviate in the other areas of this PR, since the content being outlined is also blue (or on a blue background), so a white outline makes more sense there. But in this particular case I _think_ the focus border may work well. But LMK if you disagree!